driver/src/driver/stablex_driver.rs

```diff
@@ -139,17 +137,22 @@ mod tests {
         let state = AccountState::with_balance_for(&orders);
 
         let batch = U256::from(42);
-        reader.expect_get_auction_index().return_const(Ok(batch));
+        reader
+            .expect_get_auction_index()
+            .return_once(move || Ok(batch));
```

Comment:
Is this somehow related to anyhow? is `move` something native to anyhow?

Reply:
`move` can be put in front of a closure (lambda function) to indicate that the variables it captures from the environment should be moved into the closure instead of being references. So in this case it means `bach` is moved into the closure so that the closure can return it . 
https://doc.rust-lang.org/stable/book/ch13-01-closures.html#capturing-the-environment-with-closures